### DISCUSSION TITLE
Crimson acid breaks monster design balance

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14997] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

```xml
<StatusEffect type="OnSuccess" target="Limb" duration="10.0">
  <Affliction identifier="acidburn" amount="7.5" />
  <!-- Make sure it's effective regardless of the size of the enemy, 75 damage flat and 20% health damage to an enemy without resistances -->
  <Affliction identifier="acidburn" amount="2" MultiplyByMaxVitality="true" />
  <Affliction identifier="slow" amount="20" MultiplyByMaxVitality="true" />
</StatusEffect>
```
20% max vitality acid burn means you can kill almost any monster in 5 syringe
some mod monsters are not designed to be killed easily
the poison will force modder to add acid burn immunity to their monster

### Reproduction steps

1. shoot 5 crimson acid to final boss
2. you win

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.18.1 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed since the report was created, looks like we just forgot to close the ticket. :)
